packages/frontend/tests/integration/components/pod/chat/chat-entry/emoji-test.ts

```diff
@@ -0,0 +1,88 @@
+import { module, test } from 'qunit';
+import { render, fillIn } from '@ember/test-helpers';
+import { setupRenderingTest } from 'ember-qunit';
+import { hbs } from 'ember-cli-htmlbars';
+import { getStore } from 'emberclear/tests/helpers';
+
+module('Integration | Component | chat-entry', function(hooks) {
+  setupRenderingTest(hooks);
+
+  module('emoji code replacement', function() {
+    module('there are no emoji codes to replace', function() {
+      test('result does not contain emoji', async function(assert) {
```

Comment:
because these two tests share the same setup, should that be extracted to a `beforeEach`?

Reply:
Done